shared/chat/inbox/container.js

```diff
@@ -32,26 +33,79 @@ const passesFilter = (i: Constants.InboxState, filter: string): boolean => {
 const filteredInbox = createImmutableEqualSelector(
   [getInbox, getSupersededByState, getAlwaysShow, getFilter],
   (inbox, supersededByState, alwaysShow, filter) => {
-    const ids = []
-    // Building a list using forEach for performance reason, only call i.conversationIDKey once
+    const smallIds = []
```

Comment:
we have to partition the teams into 'big' and 'small' . we have to group them first in order to do this. and after we need to resort so we need to keep the timestamps around